File: cpp/whileReverse.cpp
#include <iostream>
using namespace std;

int main()
{
    int n, t, r=0;
    cout<<"Enter any number : ";
    cin>>n;
    t=n;
    while(t>0){
        r=(r*10+(t%10));
        t=t/10;
    }
    cout<<r<<endl;
}

File: java/whileReverse.java
package git;
import java.util.Scanner;

public class whileReverse {
    public static void main(String str[]){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter any number to get it reverse : ");
        int n = sc.nextInt(), t=n, r=0;
        while(true){
            int l = t % 10;     
            t/=10;              
            r = r * 10 + l;     
            if(t==0){
                break;
            }
        }
        System.out.println("The reverse of "+n+" is "+r);
    }   
}
